homeassistant/components/honeywell/climate.py

```diff
@@ -126,31 +129,36 @@ def __init__(self, client, device, cool_away_temp,
         self._username = username
         self._password = password
 
-        self._supported_features = (SUPPORT_PRESET_MODE |
-                                    SUPPORT_TARGET_TEMPERATURE_RANGE)
+        _LOGGER.debug("latestData = %s ", device._data)  # noqa; pylint: disable=protected-access
 
-        # pylint: disable=protected-access
-        _LOGGER.debug("uiData = %s ", device._data['uiData'])
+        self.entity_id = "{}.{}_{}".format('climate', DOMAIN, device.deviceid)
```

Comment:
Should I leave this in? 

People will now have a unique, persistent ID, but they will have to search-and-replace entity IDs in automations, etc.

Reply:
Remove this and make a good name they are used to generate the initial name

Reply:
Hmmm... Looking at https://developers.home-assistant.io/docs/en/entity_registry_index.html, I don't think can satisfy everybody - I will remove this for now, as the PR definitely needs merging.